setup.cfg

```diff
@@ -0,0 +1,107 @@
+[metadata]
+name = sqlfluff
+description = The SQL Linter for Humans
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/sqlfluff/sqlfluff
+author = Alan Cruickshank
+author_email = alan@designingoverload.com
+license = MIT License
+license_files = LICENSE.md
+project_urls =
+    Homepage = https://www.sqlfluff.com
+    Documentation = https://docs.sqlfluff.com
+    Changes = https://github.com/sqlfluff/sqlfluff/blob/main/CHANGELOG.md
+    Source = https://github.com/sqlfluff/sqlfluff
+    Issue Tracker = https://github.com/sqlfluff/sqlfluff/issues
+    Twitter = https://twitter.com/SQLFluff
+    Chat = https://github.com/sqlfluff/sqlfluff#sqlfluff-on-slack
+classifiers =
+    Development Status :: 4 - Beta
+    Environment :: Console
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: Unix
+    Operating System :: POSIX
+    Operating System :: MacOS
+    Operating System :: Microsoft :: Windows
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
+    Topic :: Utilities
+    Topic :: Software Development :: Quality Assurance
+keywords =
+    sqlfluff
+    sql
+    linter
+    formatter
+    bigquery
+    exasol
+    hive
+    mysql
+    postgres
+    redshift
+    snowflake
+    spark3
+    sqlite
+    teradata
+    tsql
+    dbt
+
+[options]
+package_dir =
+    =src
+packages = find:
+python_requires = >=3.6
+install_requires =
+    # Used for finding os-specific application config dirs
+    appdirs
+    # Cached property for performance gains
+    cached-property
+    # To get the encoding of files.
+    chardet
+    click>=7.1
+    colorama>=0.3
+    configparser
```

Comment:
This module is built into Python 3, apparently. Can it be removed from here?

https://docs.python.org/3/library/configparser.html

Reply:
let me give it a go

Reply:
@barrywhart I removed it and did a fresh reinstall of the environment and the unit tests all look good so I think we should be fine with removing it